docs/doc/30-reference/30-sql/20-query-syntax/dml-join.md

```diff
@@ -0,0 +1,129 @@
+---
+title: JOIN
+---
+
+A *join* allows you to combine columns from two or more tables into a single result set. Databend supports the following *join* types:
+
```

Comment:
Better to have a general structure, like:
```sql
SELECT <expr_list>
FROM <left_table>
[GLOBAL] [INNER|LEFT|RIGHT|FULL|CROSS] [OUTER|SEMI|ANTI|ANY|ASOF] JOIN <right_table>
(ON <expr_list>)|(USING <column_list>) ...
```
From clickhouse doc

Reply:
Can we have this structure until when we support them all ?